configs/default/TMTR-TMPACERF7.config

```diff
@@ -8,37 +8,116 @@ resource MOTOR 1 C09
 resource MOTOR 2 C08
 resource MOTOR 3 B00
 resource MOTOR 4 B01
+resource MOTOR 5 B07
+resource MOTOR 6 B06
+resource MOTOR 7 NONE
```

Comment:
Remove all added entries ending with NONE as they are default.
We only need to add motor resources for motor 5 and 6 including the timer and dma settings here.

Reply:
done